spiketools/spatial/occupancy.py

```diff
@@ -33,31 +35,51 @@ def compute_spatial_bin_edges(position, bins, area_range=None):
     >>> bins = [5, 4]
     >>> compute_spatial_bin_edges(position, bins)
     (array([1. , 1.8, 2.6, 3.4, 4.2, 5. ]), array([ 6.,  7.,  8.,  9., 10.]))
+
+	Compute bin edges for an example 1d space, with positions 1 through 5.
+    >>> position = np.array([1, 2, 3, 4, 5])
+    >>> bins = [5]
+    >>> compute_spatial_bin_edges(position, bins)
+    array([1. , 1.8, 2.6, 3.4, 4.2, 5. ])
     """
 
-    _, x_edges, y_edges = np.histogram2d(position[0, :], position[1, :],
+    # 2d case
+    if (len(bins) == 2):
+        _, x_edges, y_edges = np.histogram2d(position[0, :], position[1, :],
                                          bins=bins, range=area_range)
 
-    return x_edges, y_edges
+        return x_edges, y_edges
 
+    # 1d case
+    else:
```

Comment:
As an `else`, this will catch all cases, including things that might not be 1d. At some point in the future, we could potentially have 3d cases, so it might be best to explicitly check the exact dimensions (and this also makes it clearer on what to extend if we were to add 3d later). 

For example, how about:
```
if position.ndim == 1:
    # Do 1d case
elif position.ndim == 2:
    # Do 2d case
else:
    raise ValueError('Position input should be 1d or 2d.')
```